maia/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 anyhow = "1"
 bdk = { version = "0.18", default-features = false }
 itertools = "0.10"
-maia-core = { path = "../maia-core" }
+maia-core = { git = "https://github.com/comit-network/maia", rev = "e419cb2cf19bed6ec53eaa3672408a6205a9b705", package = "maia-core" } # Pin the maia-core version to 0.1 (before breaking change)
```

Comment:
Unfortunately, we need to pin this to a particular commit, because we have already merged the breaking changes; it took me a while to overcome the error "perhaps you're using two maia-core crates" when compiling :/

In hindsight, I guess we should have bumped the version *before* the breaking changes got in. I'm still learning the innards of cargo and dependency management in Rust... 

I guess this would not have happened if we have split the repo or published the crate on crates.io.

I will amend the comment to not mention the version so it's less confusing.